### PR TITLE
change login copy to Return to Portal

### PR DESCRIPTION
**Pull request by acchiang**
literally just a text change: login page includes link that says Return to Portal if user doesnt want to login 
ezgif com-gif-maker (2)

### CODE REVIEW

File: src/pages/Login.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ export default () => {
             Continue with GitHub
           </StyledButton>
         </ButtonContainer>
-        <A href="/">Skip For Now</A>
+        <A href="/">Return to Portal</A>
```
Comment (jackyzha0) [inline src/pages/Login.js:92]: can we do a ternary on `copyText.hackathonName` for this?

Comment (jackyzha0) [reply src/pages/Login.js:92]: isn't the Skip for Now for nwHacks?

Comment (Ciuffi) [reply src/pages/Login.js:92]: We just decided to change the copy, both will be the same

Comment (acchiang) [reply src/pages/Login.js:92]: no there is only one login.js page that changes appearance based on them, so both HC and nw have this link rn 

this gives all users an option to go back to dashboard if they decide they dont' want to proceed 

if you want to remove this link here (I admit I was a bit presumptuous when i made it) i'm cool with it, just let me know if Logout should redirect to Login page or to home page

Comment (Ciuffi) [reply src/pages/Login.js:92]: + ternary probably too complicated once we run 4 hackathons..

**Review by Ciuffi — APPROVED** [review]
Could you make your PRs smaller? I don't like reviewing such huge file diffs

**Comment by acchiang** [comment]
> Could you make your PRs smaller? I don't like reviewing such huge file diffs

maybe you can include this as a retro item during your next sprint meeting
oh wait-

**Pull request merged**